migration/mig_0011_remove_fk_and_content_tables.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright © 2020 Red Hat, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package migration
+
+/*
+   migration11 removes foreign keys to rules and rule error keys, and then removes the
+   rule and rule_error_key tables because it won't be stored in database anymore
+*/
+
+import (
+	"database/sql"
+
+	"github.com/RedHatInsights/insights-results-aggregator/types"
+)
+
+var migrationClusterRuleUserFeedback = Migration{
+	StepUp: func(tx *sql.Tx, driver types.DBDriver) error {
+		var err error
+		if driver == types.DBDriverPostgres {
+			_, err = tx.Exec(`
+              ALTER TABLE cluster_rule_user_feedback DROP
```

Comment:
I'd align it with a previous line, but it's a nitpick :)

Reply:
> I'd align it with a previous line, but it's a nitpick :)

Thank you! I just uploaded a new version fixing this. It was a problem due I used spaces instead of tabs for indenting inside the string literal, so I didn't notice it on my editor